docs/src/content/docs/components/magrepack.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: magrepack
+---
+
+"Magrepack" provides players with the ability to repack magazines. It was written by Skamdrept and Badger 2-3, and originally published under the name "SESOF MagRepack."
```

Comment:
We have an integrated tag you can use for designating the previous mod. See [carrying](https://raw.githubusercontent.com/acemod/ACE-Anvil/master/docs/src/content/docs/components/carrying.mdx) for example:

Reply:
*Tried* that now.